blueprint/core/blueprint_EcmascriptEngine.cpp

```diff
@@ -350,17 +359,16 @@ namespace blueprint
 
             // Allocate a new js heap
             dukContext = std::shared_ptr<duk_context>(
-                duk_create_heap (nullptr, nullptr, nullptr, nullptr, detail::fatalErrorHandler),
-                duk_destroy_heap
-            );
+                duk_create_heap(nullptr, nullptr, nullptr, nullptr, detail::fatalErrorHandler),
+                duk_destroy_heap);
```

Comment:
Also not a huge fan of pulling this trailing comma up. But my clang-format knowledge is weak here :)

Reply:
Ah man: https://reviews.llvm.org/D33029